mackerel-plugin-nvidia-smi/nvidia_smi.go

```diff
@@ -0,0 +1,144 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	mp "github.com/mackerelio/go-mackerel-plugin-helper"
+	"os/exec"
+	"strconv"
+	"strings"
+)
+
+var queryOptions = []string{
+	"utilization.gpu",
+	"utilization.memory",
+	"temperature.gpu",
+	"fan.speed",
```

Comment:
I tested this plugin on AWS EC2 g2.2xlarge and failed to obtain metrics because `nvidia-smi` could not retrieve `fan.speed`.

```
$ ./mackerel-plugin-nvidia-smi
2016/11/18 10:09:10 OutputValues:  strconv.ParseUint: parsing "[Not Supported]": invalid syntax: 0, 0, 56, [Not Supported], 4036, 0, 4036
$ nvidia-smi --format=noheader,nounits,csv --query-gpu=utilization.gpu,utilization.memory,temperature.gpu,fan.speed,memory.total,memory.used,memory.free
0, 0, 56, [Not Supported], 4036, 0, 4036
$ nvidia-smi -q | grep -i 'fan speed'
    Fan Speed                       : N/A
```

It seems `nvidia-smi` may output NaN value like `[Not Supported]`.  So, I think it's better to skip those `[Not Supported]` value, rather than dying by parsing them.